testsuite/MDAnalysisTests/lib/test_util.py

```diff
@@ -305,13 +282,13 @@ def test_solve_1(self, universe, ag):
 
         assert_array_almost_equal(universe.atoms[:4].positions, refpos)
         assert_array_almost_equal(universe.atoms[4].position,
-                                  np.array([110.0, 50.0, 0.0]))
+                                  np.array([110.0, 50.0, 0.0]), decimal=3)
```

Comment:
This looks like a drastic decrease in precision, doesn't it? Could it be that make_whole needs to work on doubles?